src/bin/dtnd.rs

```diff
@@ -73,7 +73,7 @@ async fn main() -> std::io::Result<()> {
             Arg::with_name("discoverydestination")
                 .short("E")
                 .long("discovery-destination")
-                .value_name("DD:local_port")
+                .value_name("DD[:local_port]")
```

Comment:
actually, this is the `remote_port` isn't it? `local_port` would refer to a port on the local machine, the listening port. 
I guess in this case just `port` would also be fine.